src/libopensc/opensc.h

```diff
@@ -940,9 +946,11 @@ int sc_cancel(sc_context_t *ctx);
 /**
  * Tries acquire the reader lock.
  * @param  card  The card to lock
+ * @param  resetok If nonzero means to continue even if the card was reset
+ *	as long as it wasn't reset between individual operations
  * @retval SC_SUCCESS on success
  */
-int sc_lock(struct sc_card *card);
+int sc_lock(struct sc_card *card, int resetok);
```

Comment:
> This is currently unused but it can be used in a future code...
> I don't think we should add unused code.

I think it can be put to use soon, but nobody will use it if it is not present.
But if it absolutely can't be there without actual users then I will remove it in a respin.

Will write about card_reader_lock_obtained in the main thread as this is a general topic and
these comments are attached to sc_lock() prototype.